webhook/bigquery.py

```diff
@@ -83,12 +80,5 @@ def write_summarization_to_table(
     errors = client.insert_rows_json(
         table_name, rows_to_insert, row_ids=bigquery.AutoRowIDs.GENERATE_UUID
     )
-    if errors != []:
-        logging_client = logging.Client()
-        logger = logging_client.logger(BIGQUERY_UPSERT_LOGGER)
-        logger.log(
-            f"Encountered errors while inserting rows: {errors}", severity="ERROR"
-        )
-        return errors
 
-    return []
+    return errors
```

Comment:
Nit: Is the return type hint for this method still Sequence[Mapping]?

Reply:
Yes. `errors` is a list of dictionary objects.